chef-utils/lib/chef-utils/dsl/service.rb

```diff
@@ -97,8 +98,8 @@ def service_script_exist?(type, script)
           file_exist?("/etc/rc.d/#{script}")
         when :systemd
           file_exist?("/etc/init.d/#{script}") ||
-            ChefUtils::DSL::Introspection.has_systemd_service_unit?(script) ||
-            ChefUtils::DSL::Introspection.has_systemd_unit?(script)
```

Comment:
good example of the patter of removing the references to the class methods and using include intead